src/Resources/app/storefront/src/scss/category-extend.scss

```diff
@@ -0,0 +1,54 @@
+  /*1.3125rem*/
```

Comment:
Please format the file properly, and if the comment in first line is not needed, remove that.